src/include/openenclave/internal/calls.h

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
+// Copyright (c) Open Enclave SDK contributors.
```

Comment:
Now that we are using an OE submodule, do we actually need copies of any of these internal headers?  Can we just remove them and add a -I line pointing at the relevant places in our submodule?

Reply:
I thought so too, but it's not perfect either. E.g. when we override `OE_SDK_ROOT` to use a different OE (which I do most of my time), then that installation will usually not contain the internal headers. I think we should make OE publish what clients (i.e. us) actually need and see whether we really need all of that.